modules/gitops-deploying-a-rollout.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/getting-started-with-argo-rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-deploying-a-rollout_{context}"]
+= Deploying a rollout
+
+As a cluster administrator, you can configure Argo Rollouts to progressively route a subset of user traffic to a new application version. This allows you to test if the application is deployed and working.
+
+The following example procedure creates a `rollouts-demo` rollout and service. It then routes 20% of traffic to a canary version of the application, waits for a manual promotion, and then performs multiple automated promotions until it routes the entire traffic to the new application version.
+
+.Procedure 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators* -> *{gitops-title}* -> *Rollout*.
+
+. Create or select the project where you want to create and configure a `Rollout` custom resource (CR) from the *Project* drop-down menu.
+
+. Click *Create Rollout* and enter the following configuration in the YAML view:
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollouts-demo
+spec:
+  replicas: 5
+  strategy:
+    canary: #<1>        
+      steps: #<2>
+      - setWeight: 20 #<3>       
+      - pause: {}  #<4>         
+      - setWeight: 40 
+      - pause: {duration: 45}  #<5>     
+      - setWeight: 60
+      - pause: {duration: 20}       
+      - setWeight: 80
+      - pause: {duration: 10}
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: rollouts-demo
+  template: #<6>
+    metadata:
+      labels:
+        app: rollouts-demo
+    spec:
+      containers:
+      - name: rollouts-demo
+        image: argoproj/rollouts-demo:blue
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+        resources:
+          requests:
+            memory: 32Mi
+            cpu: 5m
+----
+<1> The deployment strategy that the rollout must use.
```

Comment:
What are valid possible values here and in the following steps?

Reply:
the value is canary that we must use here as the deployment strategy. Already it is mentioned.